packages/endpoint/package.json

```diff
@@ -3,8 +3,8 @@
   "version": "1.2.2",
   "description": "Declarative Network Interface Definitions",
   "sideEffects": false,
-  "main": "dist/index.cjs.js",
-  "module": "lib/index.js",
+  "main": "dist/index.cjs",
```

Comment:
per discussion, let's add `dist/package.json` with `"type": "commonjs"` and make this a `.js` file.